torch/csrc/jit/codegen/cuda/scheduler/normalization.cpp

```diff
@@ -43,6 +43,9 @@ ReductionParams innerPersistentHeuristic(
   // Set some targets for parallelization
   const int64_t n_elems = total_reduction_numel * total_iteration_numel;
 
+  const int64_t outer_reduction_numel =
```

Comment:
Actively working on these heuristics for cross-block persistent kernels. This file can mostly be ignored.